data/changelog.txt

```diff
@@ -227,43 +249,46 @@ Major optimization of field processing.
 Adds reachability cache/filter that can reject bresenham-based LOS checks in O(1) time.
 Uses field_cache in map::get_field to speedup fire check in npc::assess_danger.
 Speedup quicksave by limiting ui redraw rate
+Prevents slow processing of NPC overmap travel when `goal` is unreachable.
+Fixes the performance bug caused by `verify_vehicle_cache` recalculation.
+Adds lightweight crafting inventory to improve NPC construction time.
 
 ## Infrastructure:
 Adds basic infrastructure for diseases.
 Allow for specialization of activity parameters.
 Improved event transformations and statistic tracking.
 Jsonize Autodoc quality.
```

Comment:
```suggestion
Allows for specialization of activity parameters.
Improves event transformations and statistic tracking.
Jsonizes Autodoc quality.
```
"event transformations"?

Reply:
It's a bit technical, there's an event bus that broadcasts game events to several different sub-systems, and sometimes events need to be adjusted in the process.
https://github.com/CleverRaven/Cataclysm-DDA/pull/39247